--- .env/lib64/python3.10/site-packages/ray/rllib/utils/spaces/simplex.py	2024-09-05 18:50:02.242901130 +0200
+++ .env/lib64/python3.10/site-packages/ray/rllib/utils/spaces/simplex_new.py	2024-09-05 18:49:49.509916812 +0200
@@ -27,13 +27,7 @@
         super().__init__(shape, dtype)
         self.dim = self.shape[-1]
 
-        if concentration is not None:
-            assert (
-                concentration.shape == shape[:-1]
-            ), f"{concentration.shape} vs {shape[:-1]}"
-            self.concentration = concentration
-        else:
-            self.concentration = np.array([1] * self.dim)
+        self.concentration = concentration if concentration is not None else np.array([1] * self.dim)
 
     def sample(self):
         return np.random.dirichlet(self.concentration, size=self.shape[:-1]).astype(
