--- .env/lib64/python3.10/site-packages/ray/rllib/algorithms/algorithm_config.py	2024-07-31 19:21:17.559560844 +0200
+++ .env/lib64/python3.10/site-packages/ray/rllib/algorithms/algorithm_config_new.py	2024-09-18 14:26:43.209930517 +0200
@@ -3821,14 +3821,7 @@
         # If `evaluation_num_workers` > 0, warn if `evaluation_interval` is
         # None.
         if self.evaluation_num_workers > 0 and not self.evaluation_interval:
-            logger.warning(
-                f"You have specified {self.evaluation_num_workers} "
-                "evaluation workers, but your `evaluation_interval` is None! "
-                "Therefore, evaluation will not occur automatically with each"
-                " call to `Algorithm.train()`. Instead, you will have to call "
-                "`Algorithm.evaluate()` manually in order to trigger an "
-                "evaluation run."
-            )
+            pass
         # If `evaluation_num_workers=0` and
         # `evaluation_parallel_to_training=True`, warn that you need
         # at least one remote eval worker for parallel training and
