--- .env/lib64/python3.10/site-packages/ray/tune/logger/tensorboardx.py	2024-07-31 19:21:17.857560558 +0200
+++ .env/lib64/python3.10/site-packages/ray/tune/logger/tensorboardx_new.py	2024-09-18 14:35:16.998079587 +0200
@@ -45,7 +45,7 @@
             from tensorboardX import SummaryWriter
         except ImportError:
             if log_once("tbx-install"):
-                logger.info('pip install "ray[tune]" to see TensorBoard files.')
+                pass
             raise
         self._file_writer = SummaryWriter(self.logdir, flush_secs=30)
         self.last_result = None
