--- .env/lib64/python3.12/site-packages/ray/rllib/utils/debug/deterministic.py	2024-12-04 08:56:08.352374276 +0000
+++ .env/lib64/python3.12/site-packages/ray/rllib/utils/debug/deterministic_new.py	2024-12-04 17:48:43.779013597 +0000
@@ -35,7 +35,7 @@
         # See https://github.com/pytorch/pytorch/issues/47672.
         cuda_version = torch.version.cuda
         if cuda_version is not None and float(torch.version.cuda) >= 10.2:
-            os.environ["CUBLAS_WORKSPACE_CONFIG"] = "4096:8"
+            os.environ["CUBLAS_WORKSPACE_CONFIG"] = ":4096:8"
         else:
             from packaging.version import Version
 
