--- .env/lib64/python3.10/site-packages/ray/rllib/utils/debug/deterministic.py	2024-08-05 19:13:56.881303163 +0200
+++ .env/lib64/python3.10/site-packages/ray/rllib/utils/debug/deterministic_new.py	2024-08-05 19:15:26.761182729 +0200
@@ -35,7 +35,7 @@
         # See https://github.com/pytorch/pytorch/issues/47672.
         cuda_version = torch.version.cuda
         if cuda_version is not None and float(torch.version.cuda) >= 10.2:
-            os.environ["CUBLAS_WORKSPACE_CONFIG"] = "4096:8"
+            os.environ["CUBLAS_WORKSPACE_CONFIG"] = ":4096:8"
         else:
             from packaging.version import Version
 
